### PR TITLE
🌐 Add Spanish translation of Advanced User Guide - WSGI

### PR DESCRIPTION
**Pull request by bezaca**
Hi there :handshake: !
This pull request include the Spanish translation of advanced/wsgi.md

This pull request continues with the Spanish translation of #1218

### CODE REVIEW

File: docs/es/docs/advanced/wsgi.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Incluyendo WSGI - Flask, Django, entre otros 
+
+Puedes montar aplicaciones WSGI como observaste en [Sub Aplicaciones - Montajes](./sub-applications.md){.internal-link target=_blank}, [Detrás de un Proxy](./behind-a-proxy.md){.internal-link target=_blank}.
```
Comment (jovicon) [inline docs/es/docs/advanced/wsgi.md:3]: example
```suggestion
Puedes montar aplicaciones WSGI como visté en [Sub Aplicaciones - Montajes](./sub-applications.md){.internal-link target=_blank}, [Detrás de un Proxy](./behind-a-proxy.md){.internal-link target=_blank}.
```

File: docs/es/docs/advanced/wsgi.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Para esto , puedes utilizar `WSGIMiddleware` y posteriormente utilizarlo para empaquetar tú aplicación WSGI , por ejemplo , Flask, Django, etc.
```
Comment (jovicon) [inline docs/es/docs/advanced/wsgi.md:5]: Para eso
```suggestion
Para eso, puedes utilizar `WSGIMiddleware` y posteriormente utilizarlo para empaquetar tú aplicación WSGI , por ejemplo , Flask, Django, etc.
```

File: docs/es/docs/advanced/wsgi.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+```Python hl_lines="2-3  22"
+{!../../../docs_src/wsgi/tutorial001.py!}
+```
+
+## Pruébalo
+
+Ahora , cada petición que se encuentre bajo el <abbr>path</abbr> `/v1/` será manejado por la aplicación de Flask.
+
+Y el resto va ser manejado por **FastAPI**.
```
Comment (jovicon) [inline docs/es/docs/advanced/wsgi.md:23]:
```suggestion
Y el resto será manejado por **FastAPI**.
```

**Review by jovicon — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**